FAQUse: update how change anonscore target (#1171)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

::::

:::details
### How can I change the anonymity score target?
 

> The anonscore target is determined by the `Coinjoin Strategy`, but can be manually changed in the `Coinjoin Settings` dialog.

Reviewer: ```suggestion
The anonscore target is a wallet specific setting which is determined by the `Coinjoin Strategy`, but can also be manually changed in the `Coinjoin Settings` dialog.
```

> In the GUI: `Coinjoin Settings`>`Coinjoin strategy`, click on Change>`Customize`.

Reviewer: ```suggestion

On the main view click the three dots on the top right corner after loading the wallet.
Choose `Coinjoin Settings`, click `Change` (Coinjoin strategy), click `Customize`.
```

> Move the `Anonymity score target` slider to the desired amount and click Done.

Reviewer: ```suggestion
Move the `Anonymity score target` slider to the desired value and click `Done`.
```

![Coinjoin Settings Anonymity score target](/AnonymityScoreTarget.png "Coinjoin Settings Anonymity score target")

Alternatively, open the Wallet File from the wallet GUI, go to the SearchBar and click `Wallet Folder` and open the wallet json file.

```json

> "AnonScoreTarget": 50,

Reviewer: ```suggestion
"AnonScoreTarget": 5,
```

```

> Note that the anonscore target is set per individual wallet.
> 

Reviewer: ```suggestion
```

::::

:::details
### How can I change the anonymity score target?
 
The anonscore target is a wallet specific setting which is determined by the `Coinjoin Strategy`, but can also be manually changed in the `Coinjoin Settings` dialog.

On the main view click the three dots on the top right corner after loading the wallet.
Choose `Coinjoin Settings`, click `Change` (Coinjoin strategy), click `Customize`.
Move the `Anonymity score target` slider to the desired value and click `Done`.

![Coinjoin Settings Anonymity score target](/AnonymityScoreTarget.png "Coinjoin Settings Anonymity score target")

> Alternatively, open the Wallet File from the wallet GUI, go to the SearchBar and click `Wallet Folder` and open the wallet json file.

Reviewer: ```suggestion
Alternatively, you can change the anonymity score target by editing the wallet file.
Go to the SearchBar and click `Wallet Folder` and open the wallet json file and edit the `AnonScoreTarget` value.
```

::::

:::details
### How can I change the anonymity score target?
 
The anonscore target is a wallet specific setting which is determined by the `Coinjoin Strategy`, but can also be manually changed in the `Coinjoin Settings` dialog.

On the main view click the three dots on the top right corner after loading the wallet.
Choose `Coinjoin Settings`, click `Change` (Coinjoin strategy), click `Customize`.
Move the `Anonymity score target` slider to the desired value and click `Done`.

![Coinjoin Settings Anonymity score target](/AnonymityScoreTarget.png "Coinjoin Settings Anonymity score target")

Alternatively, you can change the anonymity score target by editing the wallet file.
Go to the SearchBar and click `Wallet Folder` and open the wallet json file and edit the `AnonScoreTarget` value.

```json
"AnonScoreTarget": 5,
```

> 

Reviewer: ```suggestion
```

> @[youtube](gWo2RAkIVrE)

Reviewer: I am removing this video, it has nothing to do with changing the anonymity score target.

```suggestion
```
